### DISCUSSION TITLE
concurrent connection

### DISCUSSION THREAD

**Discussion by sindhubha** [fastapi/fastapi#9882] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Fastabi server has hang problem due to concurrent connection
```

### Description

Fastabi server has hang problem due to concurrent connection 

 kindly give me the solution for  that simultaneous  connection...

### Operating System

Windows

### FastAPI Version

0.95.1

### Python Version

3.9.12

**Comment by iudeen** [comment]
Do you have a code we can see?
